plugin-ci/orb.yml

```diff
@@ -104,11 +104,22 @@ jobs:
       - checkout
       - *restore_cache
       - run: make dist
+      - run:
+          name: "Generating release notes"
+          command: |
+            echo "### Supported Mattermost Server Versions: $(cat plugin.json | jq .min_server_version -r)+"  >> release-notes.md
```

Comment:
Is `jq` installed by default?

Reply:
Yes it is in the `circleci/golang:1.13-node` image.